packages/ui/src/Toggle.module.scss

```diff
@@ -0,0 +1,98 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+$h: 24px;
+$w: 60px;
+$knob-r: $w / 2 + 8px;
```

Comment:
nit: Should we use cameCase for consistency with other components?

Reply:
done. (wish it was kebab case all around but too late :/)